.github/workflows/build.yml

```diff
@@ -1,13 +1,8 @@
 name: build
 
 on:
-  push:
-    branches: [main, develop]
-    paths-ignore:
-      - "docs/"
-      - "**.md"
   pull_request:
-    branches: [main, develop]
+    types: [opened, synchronize]
```

Comment:
These types are not necessary as the default are: `opened`, `synchronize`, and `reopened`; according to https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request.

```suggestion
```

Reply:
What do you mean exactly?

Reply:
The default options for types are `opened`, `synchronize`, and `reopened`, which make sense for the project. So, we can remove the types parameter altogether. That would only add the `reopened` type, which makes sense.

Reply:
Ah I see, fixed!